### DISCUSSION TITLE
There is no data in web after restart tomcat or docker whithout  no code change and reindex

### DISCUSSION THREAD

**Discussion by menggya** [oracle/opengrok#4241] [General]
There is no data in web after restart tomcat or docker, but  data and config is no change, so can i just upload configuration.xml ?

There was an error!
The source root path has not been configured! Please configure your webapp.

Unable to determine source root path. Missing configuration?

java.io.FileNotFoundException: Unable to determine source root path. Missing configuration?

**Comment by vladak** [comment]
could you attach full logs from the container ? (`docker logs <container_name>`)
